### ISSUE TITLE
Upgrading to v0.3.2 results in `PhantomJS not found.`

### ISSUE THREAD

**Issue by uglymunky** [busterjs/grunt-buster#33]
I noticed these changes in the changelist which make me think this is expected but I'm not sure what I should do:

```
* Use resolve-bin to find buster and phantomjs executables
* Add buster as a peer dependency
```

**Comment by dominykas** [comment]
Duplicate of https://github.com/busterjs/grunt-buster/issues/31

**Comment by dominykas** [comment]
Published as 0.4
